### DISCUSSION TITLE
How to open a Flyout from a DropDownButton via code

### DISCUSSION THREAD

**Discussion by tonka3000** [bdlukaa/fluent_ui#718] [Q&A] [answered]
Hi,

I try to open a flyout menu from a `DropDownButton` via code. The button works nicely when I click on it via mouse but I want to show the menu also when a press a shortcut on the keyboard. Regardless of what I'm trying I could not find a way to open it.

I already added a FyloutController, but I'm not sure if this is correct, because the flyout controller seems not to have a simple open method or something.

```dart
 return FlyoutTarget(
          controller: widget.controller ?? controller,
          child: DropDownButton(
              key: widget.buttonKey,
              title: const Text('My Dropdown'),
              items: actions
                  .map((e) => MenuFlyoutItem(
                      text: Text(e.getTitle()),
                      onPressed: () {
                        e.execute();
                      }))
                  .toList()));
```

Can somebody give me some advice how to do that (maybe there is a better way to do it via shortcuts/actions/intent).

Greetings
Michael

**Comment by bdlukaa** [comment, marked as answer]
#723
